components/SocialSharing/SocialSharing.tsx

```diff
@@ -40,6 +41,7 @@ export const SocialSharing = ({
   };
 
   const safeUrl = encodeURIComponent(`${config.siteUrlProd}/${slug}`);
+  const safeTitle = encodeURIComponent(title);
```

Comment:
Wrap title in encodeURIComponent to convert the white spaces and any special characters